src/gen/stream_consumer/v1alpha1/stream_consumer_grpc_pb.d.ts

```diff
@@ -0,0 +1,268 @@
+// package: stream_consumer.v1alpha1
```

Comment:
Is it auto-generated? Could we only commit the sources and add generated files to `.gitignore`?
Example: https://github.com/proxima-one/state-manager/tree/master/client-ts

Reply:
How long does the generation take? We discussed with Ivan that if it is time-consuming, we can leave those files commited. But if it's unnoticeable I think we better make generation a build step.

Reply:
It can be autogenerated and ignored